src/AppInstallerCLICore/Commands/PinCommand.cpp

```diff
@@ -0,0 +1,244 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "PinCommand.h"
+#include "Workflows/CompletionFlow.h"
+#include "Workflows/PinFlow.h"
+#include "Workflows/WorkflowBase.h"
+#include "Resources.h"
+
+namespace AppInstaller::CLI
+{
+    using namespace AppInstaller::CLI::Execution;
+    using namespace AppInstaller::Utility::literals;
+    using namespace std::string_view_literals;
+
+    // TODO: Create this link with the docs!
+    static constexpr std::string_view s_PinCommand_HelpLink = "https://aka.ms/winget-command-pin"sv;

```

Comment:
@denelon Can you create this even if it points to just the general documentation (or add a note for you to do that)? I remember with a previous command I added it we completely forgot about it...

Reply:
If I remember correctly, aka.ms will just default somewhere else if the URL isn't registered.
Yes, we can create the aka.ms links ahead of time with a temporary/placeholder URL and modify it at any point later.

Reply:
Aka.ms links that don't exist just redirect to the Bing homepage iirc

Reply:
Yeah, they redirect to the Bing homepage. I was mentioning creating it now pointing to the general documentation, because if we do forget about creating the right link, in my mind it's better if the link at least points to somewhere close instead of going to Bing.

Reply:
I added https://aka.ms/winget-command-pin

It currently defaults to the main winget page.